README.md

```diff
@@ -40,8 +40,8 @@ express-fastest-validator uses [fastest-validator](https://github.com/icebob/fas
 const loginSchema = {
   body: {
     username: { type: "string", min: 3, max: 15 },
-    password: { type: "string", min: 8 },
-  },
+    password: { type: "string", min: 8 }
+  }
 };
```

Comment:
Soon I'll update the repo with eslint configs until then refrain from committing linting fixes